testing/marketplace/deployer_envsubst_base/standard/test

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -eox pipefail
+
+test_directory=$(mktemp --dry-run)
+cp -r $(dirname "$0") "$test_directory"
+cd "$test_directory"
+
+image="$REGISTRY/testing/deployer_envsubst/1:$MARKETPLACE_TOOLS_TAG-$TEST_ID"
+
+docker build \
+    --build-arg "MARKETPLACE_TOOLS_TAG=$MARKETPLACE_TOOLS_TAG" \
+    --tag "$image" \
+    .
+
+docker push "$image"
+
+docker run \
+    "gcr.io/cloud-marketplace-tools/k8s/dev:$MARKETPLACE_TOOLS_TAG" \
+    cat /scripts/dev \
+  > dev
+chmod a+x dev
+
+export MARKETPLACE_TOOLS_IMAGE="gcr.io/cloud-marketplace-tools/k8s/dev"
+
+app_parameters="{}"
+./dev \
+  /scripts/verify \
+      --deployer="$image" \
+      --parameters="$app_parameters"
```

Comment:
use single quotes--we were hit by this before

Reply:
Single quotes were necessary in Makefile, which preprocess quotes differently.

Using single quotes here will cause the string literal $app_parameters to be passed to verify.